adapter_nrf51.go

```diff
@@ -47,6 +47,7 @@ func handleEvent() {
 		switch id {
 		case C.BLE_GAP_EVT_CONNECTED:
 			currentConnection.Reg = gapEvent.conn_handle
+			DefaultAdapter.connectHandler(Address{}, true)
```

Comment:
Because this is an interface, you could consider passing nil. That makes it very clear that this address is not set (and is not a hypothetical 00:00:00:00:00:00 MAC address). It also avoids a possible heap allocation (I haven't checked it, but I suspect it would otherwise cause a heap allocation in an interrupt).

The same goes for other `connectHandler` events.

Reply:
All updated.